src/MBrace.Streams/CloudStreams.fs

```diff
@@ -692,3 +692,16 @@ module CloudStream =
         }
 
 
+
+    // Text ReadLine CloudFile producers
+
+    /// <summary>
+    /// Constructs a CloudStream from a collection of CloudFiles using text line reader.
+    /// </summary>
+    /// <param name="sources">The collection of CloudFiles.</param>
+    let ofTextCloudFiles (sources : seq<CloudFile>) : CloudStream<string> =
```

Comment:
The name feels ambiguous - I suggest name ``ofCloudFilesByLines``?  This makes room for ``ofCloudFilesByText`` and ``ofCloudFilesByXmlChunk`` etc.